assets/css/stripeStyle.css

```diff
@@ -266,6 +260,192 @@ table tr {
 	color: #304ffe;
 }
 
+.toggle {
+	display: none;
+}
+
+.mobile-price-view {
+	display: none;
+}
+
+.mobile-price-view.opened {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	position: fixed;
+	top: 0;
+	width: 100%;
+	height: 100vh;
+	background: transparent linear-gradient(233deg, #d500f9 0%, #2962ff 99%) 0% 0% no-repeat padding-box;
+	z-index: 100;
+	margin-left: -1rem;
+	padding: 1rem;
+}
+
+.mobile-price-view .nav-back {
+	display: flex;
+	align-items: center;
+}
+
+.mobile-price-view .nav-back svg {
+	margin-right: 10px;
+}
+
+
+.mobile-price-view .nav-back span {
+	padding: 1rem;
+	padding-left: 0;
+	font: 600 16px/22px "Open Sans", sans-serif;
+}
+
+.mobile-price-view .disclaimer {
+	margin-top: auto;
+}
+
+/* Desktop */
+@media screen and (min-width: 1024px) {
+
+	.columns {
+		display: flex;
+		flex-direction: row;
+	}
+
+	.main.columns {
+		height: 100vh;
+		overflow: hidden;
+	}
+
+	.column {
+		width: 50%;
+	}
+	
+	.wrapper-form.column {
+		width: 67%;
+	}
+
+
+	.price-column {
+		width: 33%;
+	}
+
+	.button-bar {
+		width: 67%;
+	}
+
+	.mobile-price-info {
+		display: none;
+	}
+	
+}
+
+/* Tablet */
+@media screen and (max-width: 1023px) {
+
+	.column {
+		width: 100%;
+	}
+
+	.summary .columns {
+		display: flex;
+		flex-direction: row;
+	}
+	
+
+	.summary .column {
+		width: 50%;
+	}
+
+	.wrapper-form.column {
+		width: 100%;
+	}
+
+	.price-column {
+		width: 100%;
+	}
+
+	.button-bar {
+		width: 100%;
+	}
+
+	.wrapper-form {
+		padding-bottom: 100px;
+	}
+
+	.mobile-price-info {
+		display: block;
+		display: flex;
+    	align-items: center;
```

Comment:
Codacy found an issue: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)